recipe/meta.yaml

```
@@ -22,7 +22,7 @@ requirements:
     - gnuconfig
     - automake
     - pkg-config
-    - xorg-util-macros
+    - xorg-util-macros >=1.8
   host:
     - xorg-kbproto
```

```suggestion
    - xorg-xorgproto
```
I think the only reason this is working is that it's pulling in `xorg-xorgproto: 2024.1-h5eee18b_0` instead of b1. We might need to hotfix b0 as well to help prevent these packages from coexisting.